### ISSUE TITLE
libevel support for PNPRegistration event

### ISSUE THREAD

**Issue by Rohit2387** [att/evel-library#26]
Hi,

whether libevel support PNPRegistration Event?

**Comment by PelagicGames** [comment]
I don't think so. There's no reference to PNP/PNF in the repository, and it hasn't been updated since May 2017.
